Assets/Scripts/Controllers/MouseController.cs

```diff
@@ -580,6 +570,7 @@ private void ShowFurnitureSpriteAtTile(string furnitureType, Tile t)
         sr.sprite = fsc.GetSpriteForFurniture(furnitureType);
 
         if (WorldController.Instance.World.IsFurniturePlacementValid(furnitureType, t) &&
+            WorldController.Instance.World.IsFurnitureWorkSpotClear(furnitureType, t) && 
```

Comment:
I was of the opinion that it is conceptually separate from if the placement of the actual furniture is valid, so separated it out.